cadc-storage-permissions/src/test/resources/valid.properties

```diff
@@ -0,0 +1,16 @@
+# Sample properties file for cadc-storage-permissions.
```

Comment:
Although this is what we talked about, now that I see these examples I think it is pretty messy in a few cases:

Any pattern with anon = T doesn't need any read-only groups so ops has to config a dummy group.
Anything beyond one group is easy to mess up syntactically... maybe we need something that allows null values so only positive (granting) config is required. TBD